### DISCUSSION TITLE
batch file doesnt' works

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#5105] [Help]
Hello,
I have a batch file with this content : 
magick out.tif -crop 72x74+319+92 +repage ftxt: > cielab.txt

and the content of the cielab.txt file is : 

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:  

C:\Users\Sylvain ARD\Dropbox\Documents Sylvain\autoentrepreneur\Contrats\Projet couleurs\PROGRAMME\Delphi\Win32\Debug>magick out.tif -crop 72x74+319+92 +repage ftxt:

infinite looping

please help !

**Comment by urban-warrior** [comment]
This appears to be a Windows batch issue as opposed to an issue with ImageMagick.  Bring up a Command Prompt window and issue the comand:

```
magick out.tif -crop 72x74+319+92 +repage ftxt: > cielab.txt
```

We assume the command completes without looping.  If so, your issue is with a batch file and you may want to find another community that can assist you with Windows batch.

**Comment by SylvainArd** [comment]
thank you, I asked to a Batch forum too, this comes with differents
commands but not all
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers

**Comment by snibgo** [comment]
What version of IM do you use?

What is the name of your batch file? What exactly does it contain?

How do you run the batch file? I suppose you type something at the command line. What exactly do you type?

**Comment by SylvainArd** [comment]
my version of IM is 7.1.0-30-Q8
the name of my batch file is magick.bat
I run the batch file by double-clicking on it
the batch file contains only :

magick out.tif -crop 72x74+319+92 +repage ftxt: > cielab.txt

Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers

**Comment by SylvainArd** [comment]
I found the problem, my batch file was called magick.bat so it called
itself infinitively, I renamed it and it works
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers
